callstack/react-native-pager-view
# fix(android): fix building fabric example

# Summary

This pull request fixes issue causing `./gradlew assembleRelease` to throw an error on **Fabric** architecture example.

This was caused by incorrect `index` file extension, which has been changed from `.ts` to `.js` to match legacy arch example.

## Test Plan

1. navigate into `fabricexample/android` directory
2. run `./gradlew assembleRelease`
3. app should build without errors

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |

## Checklist
